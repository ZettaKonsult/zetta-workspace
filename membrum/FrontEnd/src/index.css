:root {
  --primary: #337ab7;

  --danger: #f44336;
  --danger-light: #ffdddd;

  --success: #4caf50;
  --success-light: rgb(200, 255, 200);

  --info: #2196f3;
  --info-light: #e7f3fe;

  --warning: #ff9800;
  --warning-light: #ffffcc;
}

html,
body {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: sans-serif;
}

#root {
  height: 100%;
}

.ButtonGroup > button {
  margin: 0.5em;
}

.InputGroup {
  display: flex;
  flex-flow: column nowrap;
  margin: 1em 2em;
  justify-content: center;
}

.InputGroup > div {
  margin: 0.5em;
}
