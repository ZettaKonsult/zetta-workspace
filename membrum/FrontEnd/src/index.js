import React from 'react'
import ReactDOM from 'react-dom'
import {BrowserRouter as Router} from 'react-router-dom'
import {Provider} from 'react-redux'
import registerServiceWorker from './registerServiceWorker'

import configureStore from './configureStore'

import Routes from './Routes'
import App from './containers/App'

import './index.css'
import 'normalize.css'

let store = configureStore()

ReactDOM.render(
  <Provider store={store}>
    <Router>
      <App>
        {process.env.NODE_ENV !== 'production' ? <h3>Dev</h3> : ''}
        <Routes />
      </App>
    </Router>
  </Provider>,
  document.getElementById('root')
)

registerServiceWorker()
