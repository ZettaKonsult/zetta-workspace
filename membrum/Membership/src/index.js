/* @flow */

/**
 * @date 2018-01-17
 */

export { getSubscriptions, saveSubscriptions } from './plan';
