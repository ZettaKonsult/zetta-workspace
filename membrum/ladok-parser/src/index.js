import ladokParser from 'ladokParser'
import unionAssigner from 'unionAssigner'

module.exports = {
  getAssignments: unionAssigner.getAssignments,
  parseLadokFile: ladokParser.parse,
  parseLadokDirectory: ladokParser.parseDirectory
}
