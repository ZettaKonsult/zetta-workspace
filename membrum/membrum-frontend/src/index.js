import React from "react"
import ReactDOM from "react-dom"
import { BrowserRouter as Router } from "react-router-dom"
import registerServiceWorker from "./registerServiceWorker"
import Root from "./containers/Root"

import "./index.css"
import "normalize.css"

ReactDOM.render(
  <Router>
    <Root />
  </Router>,
  document.getElementById("root")
)
registerServiceWorker()
