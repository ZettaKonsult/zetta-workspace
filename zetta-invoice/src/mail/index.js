/* @flow */

/**
 * @date 2018-03-12
 */

import { sendInvoice } from './mail';

export { sendInvoice };
