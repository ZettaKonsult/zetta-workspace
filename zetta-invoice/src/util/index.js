import parser from './parser';
import { getS3Object, putS3Object } from './s3';
import db, { getDbTable } from './database';
import { failure, success } from './response';
import request from './http';
import testConfig from './testConfig';

export {
  parser,
  request,
  getS3Object,
  putS3Object,
  db,
  getDbTable,
  failure,
  success,
  testConfig,
};
