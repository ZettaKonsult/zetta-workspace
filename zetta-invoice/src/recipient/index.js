/* @flow */

import { getBySSN } from './get';
import { db, getDbTable } from '../util';
import recipientDatabase from './recipientDatabase';

const TableName = getDbTable({ name: 'Recipients' });
const database = recipientDatabase(db)(TableName);

export default companyCustomerId => database(companyCustomerId);
export { getBySSN };
