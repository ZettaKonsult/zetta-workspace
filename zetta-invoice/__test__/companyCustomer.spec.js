/**
 * @date 2018-03-05
 */

import { request, testConfig } from '../src/util/';

const host = testConfig.Host;

describe('companyCustomer E2E', () => {
  let companyCustomer;

  it('create', async () => {
    const data = {
      companyCustomer: {
        address: 'Road 234A',
        city: 'RecipientCity',
        email: 'firstName@recipient.com',
        firstName: 'RecipientFirst',
        lastName: 'RecipientLast',
        mobile: '+46762345678',
        ssn: '1234567890',
        zipcode: '12345',
        company: 'zetta konsult',
        vat: '91050400356',
        bank: {
          giro: '123-4567',
          name: 'MoneyBank',
        },
      },
    };

    companyCustomer = await request({
      host,
      path: 'companycustomer',
      payload: {
        method: 'post',
        body: data,
      },
    });

    expect(companyCustomer).toMatchSnapshot();
  });

  it('get', async () => {
    const result = await request({
      host,
      path: `companycustomer`,
      payload: {
        method: 'get',
      },
    });

    expect(result).toEqual(companyCustomer);
  });

  it('update', async () => {
    const result = await request({
      host,
      path: 'companycustomer',
      payload: {
        method: 'post',
        body: { companyCustomer: { ...companyCustomer, zipcode: '54321' } },
      },
    });

    expect(result).toMatchSnapshot();
  });

  it('delete', async () => {
    const data = {
      companyCustomerId: companyCustomer.id,
    };
    const result = await request({
      host,
      path: `companycustomer`,
      payload: {
        method: 'delete',
        body: data,
      },
    });

    expect(result).toEqual(true);
  });
});
