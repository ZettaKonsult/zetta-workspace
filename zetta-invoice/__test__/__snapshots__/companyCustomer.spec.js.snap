// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`companyCustomer E2E create 1`] = `
Object {
  "address": "Road 234A",
  "city": "RecipientCity",
  "company": "zetta konsult",
  "email": "firstName@recipient.com",
  "firstName": "RecipientFirst",
  "id": "offlineContext_cognitoIdentityId",
  "lastName": "RecipientLast",
  "mobile": "+46762345678",
  "vat": "91050400356",
  "zipcode": "12345",
}
`;

exports[`companyCustomer E2E update 1`] = `
Object {
  "address": "Road 234A",
  "city": "RecipientCity",
  "company": "zetta konsult",
  "email": "firstName@recipient.com",
  "firstName": "RecipientFirst",
  "id": "offlineContext_cognitoIdentityId",
  "lastName": "RecipientLast",
  "mobile": "+46762345678",
  "vat": "91050400356",
  "zipcode": "54321",
}
`;
